src/styles/App.css

```diff
@@ -1,16 +1,36 @@
 body {
-    background-color: #99dacc;
-    font-family: Helvetica, sans-serif;
+    background-color: #e2e2e2;
   }
   
   .App {
     text-align: center;
     padding-bottom: 85px;
     font-family: 'Poppins', sans-serif;
   }
+
+  .Font {
+    font-family: 'Poppins', sans-serif;
+  }
   
 
   a.Recipe-link {
     color:darkslategrey;
-    padding-top: 75%;
-  }
\ No newline at end of file
+  }
+
+  .container img {
+    width: 100%;
+ }
+
+ .MuiGrid-spacing-xs-1 {
+  width: calc(100% + 8px);
+  margin: -4px;
+  justify-content: center;
+}
+
+.MuiPaper-elevation1 {
+    box-shadow: 2px 4px 20px 0px rgb(0 0 0 / 50%), 0px 1px 1px 0px rgb(0 0 0 / 14%), 0px 1px 3px 0px rgb(0 0 0 / 12%)!important;
+}
+
+.MuiPaper-root {
+  background-color: #ffffff!important;
```

Comment:
try not to over rely too much on the `!important` rule - its usually bad practice to have these in your code and should only be used for cases like utility classes (for example - a class u use to hide an element like .hidden).

if you have too many !important in ur code, the use case for utility classes becomes moot